ruby/ql/test/library-tests/ast/ValueText.expected

```diff
@@ -10,6 +10,8 @@ exprValue
 | calls/calls.rb:26:7:26:7 | 1 | 1 | int |
 | calls/calls.rb:36:9:36:11 | 100 | 100 | int |
 | calls/calls.rb:36:14:36:16 | 200 | 200 | int |
+| calls/calls.rb:226:1:228:3 | nil | nil | nil |
```

Comment:
The locations of the synthesized nodes do not look right to me. It looks like they are inherited from the entire for statement. I think it would be better to let them inherit the location of the corresponding variable in the for-loop pattern.

Reply:
Good idea.